source/Variant.cpp

```diff
@@ -167,6 +198,24 @@ Variant Variant::parse(const std::string& s)
     if (bstr == "FALSE") {
         return Variant(false);
     }
+	
+	// check if it is a vector
+	if (str[0] == '[') {
+		string vstr = str.substr(1, str.size() - 1);
+		vector<string> parts(splitString(vstr, ","));
+
+		// check if it is a double vector
+		const char* input = parts[0].c_str();
+		char* end = 0;
+		double d = strtod(input, &end);
+		if (*input != '\0' && end != input && *end == '\0')
```

Comment:
I don't think the element of string vector will be wrapped by a quotation mark. Since we use string vectors for addReaction() and allow stoichiometry number added before species name, checking if the first character is a number will not work either. So I think the only way is to try to parse the first element and see if it is a double type.

Reply:
Vector string is removed from Variant.